frontend/wallet/package.json

```
@@ -33,7 +33,7 @@
     "pack": "fpack --development ./lib/js/src/render/Index.js",
     "query": "send-introspection-query http://localhost:8080/graphql",
     "query-fake": "concurrently --kill-others 'yarn run fake' 'sleep 5 && yarn run query' || true",
-    "reformat": "bsrefmt --in-place src/**/*.re",
+    "reformat": "bsrefmt --in-place $(find src -name '*.re' -or -name '*.rei' -print)",
```

Not sure if windows has find, but either way I think this is equivalent to 
```suggestion
    "reformat": "bsrefmt --in-place src/**/*.re{,i}",
```

---

the problem was people (and CI) didn't have globstar enabled or their bash was too old. I would doubt that windows supports either of these approaches :o